ODIN_II/SRC/simulate_blif.cpp

```diff
@@ -1189,25 +1202,27 @@ nnode_t** get_children_of(nnode_t* node, int* num_children) {
         nnet_t* net = pin->net;
         if (net) {
             /*
-             *  Detects a net that may be being driven by two
-             *  or more pins or has an incorrect driver pin assignment.
+             *  Detects a net that has an incorrect driver pin assignment.
```

Comment:
in this perticular case you do want to forloop otherwise you are doing the check twice, this one here is to sanitize the back and forward pointers are both aligned